script.service.janitor/resources/language/Dutch/strings.po

```diff
@@ -0,0 +1,430 @@
+# XBMC Media Center language file
```

Comment:
Would you be kind to (in adition to changing `en` to `en_GB`) also adapt the new folder structure for language files? See here an example: https://github.com/xbmc/generator-kodi-addon/tree/master/generators/app/templates/resources/language